src/include/btree_inline.h

```diff
@@ -2055,6 +2055,27 @@ __wt_page_swap_func(WT_SESSION_IMPL *session, WT_REF *held, WT_REF *want, uint32
     return (ret);
 }
 
+/*
+ * __wt_bounds_early_exit --
+ *     Performs bound comparison to check if the key is within bounds, if not, increment the
+ *     appropriate stat, early exit, and return WT_NOTFOUND.
+ */
+static inline int
+__wt_bounds_early_exit(
```

Comment:
Rename to __wt_btcur_bounds_early_exit so its consistent with the below function.